test/e2e/common_test.go

```diff
@@ -140,7 +146,7 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	// Cache images
 	cwd, _ := os.Getwd()
 	INTEGRATION_ROOT = filepath.Join(cwd, "../../")
-	podman := PodmanTestSetup(os.TempDir())
+	podman := PodmanTestSetup(GinkgoT().TempDir())
```

Comment:
right here

Reply:
That I saw but I fail to understand why worked before.  We did a `RemoveAll` before that is failing now.

I don't want to bother much but am trying to put the puzzle in my head together.

Reply:
No we did never call RemoveAll() on that path. This path right here was always leaked, just check /tmp for root, runroot directories. And yes RemoveAll() will work but only as long as you do not pull images with uid > 0.

Reply:
D'oh!   Thanks!